Land Jebb's additions!

towerofnix opened the pull request:
A bunch of them. We've still gotta review these.

We also cherry-picked (some of) these commits in reverse order, so... sorry about that!

Merge with hsmusic/hsmusic-media#149.
Closes #368.

JebbJabroni committed:
Added Battle of the Bottoms OST

JebbJabroni committed:
Added JadeRoute

JebbJabroni committed:
Added Acousmatic Recollection Layer EP: A corru.observer Fan Album

JebbJabroni committed:
Added dbnet18 albums and group

Albums:
- Knife to meet you
- Background noise

JebbJabroni committed:
Added Swagazaki aliases and links

JebbJabroni committed:
Changed Imperial March's location

JebbJabroni committed:
Added Omegalodon and Songs for Rune

towerofnix committed:
fix misplaced rj lake group

towerofnix committed:
general rebase fixup

JebbJabroni committed:
Added Erik Scheele group

Add description

JebbJabroni committed:
Added Pascal van den Bos group and albums

towerofnix committed:
george mccrae

towerofnix committed:
blue-room url fix

JebbJabroni committed:
Added 413 AC OST: Act 1 Side 1

Note: add tags later

JebbJabroni committed:
Added bg for Waiting For The Night (Svix Mix)

JebbJabroni committed:
Added Sburban single

JebbJabroni committed:
Added De Novo

Note: add character tags

JebbJabroni committed:
Added Songs for Rune bg

JebbJabroni committed:
Added albums for Erik Scheele group

JebbJabroni committed:
Added Marcy Nabors albums and group

JebbJabroni committed:
Added Marcy Nabors group to Vivid Spectrum

JebbJabroni committed:
Added Sean Gorter links to Flare/Sean Gorter

JebbJabroni committed:
Added SBURB Beta and Beneath The Shade by pomabug

Thanks to pomabug for sending the background assets for Beneath the Shade and Sburb Beta

JebbJabroni committed:
Added Shallow Shores by Circlejourney

JebbJabroni committed:
Added Mockat discography

- Tapestry vol 1
- Tapestry vol 1.5
- Tapestry vol 2
- Bichrome Phantasia

JebbJabroni committed:
Updated rerollroxy's name to baconeggndee

JebbJabroni committed:
Added Perfectly Generic Album by chaoticComposer

JebbJabroni committed:
Additional baconeggndee fixes

JebbJabroni committed:
Added bandcamp link for Gryotharian

JebbJabroni committed:
Added Cokestuck Vol.1

towerofnix committed:
quick jebb compat things

JebbJabroni committed:
Added CLUBSTUCK and CLUBSTUCK^2: TWERKABLY CANON

Note: should later change tag from pistolkind to clover's pistol

JebbJabroni committed:
Added 04-13-2023 Beta Kid Themes (Op. 6)

JebbJabroni committed:
Added 4-13-2025 Homestuck Beta Kid Themes (Op. 12)

JebbJabroni committed:
Added Curtain Call - Maragi Radi

JebbJabroni committed:
Added Hivebreak: Vol. 1

JebbJabroni committed:
Added Curtain Call - Maragi Radi to the Desynced group

JebbJabroni committed:
Added CLUBSTUCK artists

JebbJabroni committed:
Added CLUBSTUCK references

JebbJabroni committed:
Added Hivebreak Vol 1 references

JebbJabroni committed:
Added D

JebbJabroni committed:
Added Hivebreak special thanks

JebbJabroni committed:
Added several rj lake albums

- Muddledash!
- Mr. Bucket Told Me Tunes
- dog music
- witchstranded
- PURGATORY DUNGEONEER

towerofnix committed:
quick jebb compat (wishful-thinking-kindred-souls)

towerofnix committed:
whitespace bandaid

towerofnix committed:
omegalodon + songs for rune review

towerofnix committed:
rj lake new albums into series

Lilithtreasure committed:
muddledash! bonus tracks

Lilithtreasure committed:
main album track section, whitespace removal for squidmaster p lyrics

Lilithtreasure committed:
unsure where to put this brief commentary on the tentacles sequel; placed in main album commentary for now

towerofnix committed:
knife to meet you, background noise review

towerofnix committed:
acousmatic recollection layer ep review

towerofnix committed:
preliminary muddledash stuff

towerofnix committed:
muddledash bonus tracks review

Lilithtreasure committed:
from the top, this should affect only the muddledash yaml (ANSI to UTF-8 so this darn fancy apostrophe character can work, from ~flip tape over~)

towerofnix committed:
jaderoute, battle of the bottoms review

towerofnix committed:
pascal deltarune remixes art tags

PhantomRuby commented on art-tags/media/undertale-and-deltarune.yaml:
```diff
@@ -515,6 +530,27 @@ Color: '#FFFFFF'
+Direct Descendant Tags:
+- Locations in the closet Dark World
+---
+Tag: Locations in the closet Dark World
```
the castle town (closet) and card kingdom (abandoned classroom) are separate dark worlds

maybe dark world stuff could be structured like

- Dark Worlds
- - Card Kingdom
- - - Field
(idk, aside from the song name, the field is just called the field i believe, [the wiki sure calls it such](https://deltarune.wiki/w/Field))

towerofnix replied:
"Field" (and "Forest") look to be via save point location names: https://youtu.be/35h5NjXjoo8?t=1135

towerofnix committed:
try out "astandalone characters by particular people" tag

towerofnix committed:
jebb pascal additions review, mission to saturn single

towerofnix committed:
deltarune locations 2 re: ruby review

towerofnix committed:
squash all rebase/jebb changelog changes so far into one

towerofnix committed:
413 ac ost: act 1 side 1 review

towerofnix committed:
waiting for the night background changelog

towerofnix committed:
sburban review

towerofnix committed:
de novo review

towerofnix committed:
erik scheele group description & series

towerofnix committed:
mostly trim sean gorter not-"flare" links

towerofnix committed:
sburb beta & beneath the shade review

towerofnix committed:
shallow shores review + changelog

towerofnix committed:
the tapestry review + changelog

towerofnix committed:
monckat group description

towerofnix committed:
perfectly generic album (chaoticComposer) review + changelog

towerofnix committed:
cokestuck vol 1 review + changelog

towerofnix committed:
cokestuck vol1 artist commentary

towerofnix committed:
clubstuck artwork origin details

towerofnix committed:
clubstuck review + changelog

towerofnix committed:
casualclassical beta kid themes review + changelog

towerofnix committed:
curtain call review + changelog

dizzims. -> dizzims

towerofnix committed:
hivebreak vol 1 review + changelog

Tentative review anyhow, this does not touch RBH.

towerofnix committed:
paradox struck review + changelog

Not really anything but art tags for this one

towerofnix committed:
rj lake various review

towerofnix committed:
marcy group description

towerofnix committed:
The Castle Depths

ruby
https://discord.com/channels/749042497610842152/779895315750715422/1458614269301100554

towerofnix committed:
Cara Mia Addio!

ruby
https://discord.com/channels/749042497610842152/779895315750715422/1458614269301100554

towerofnix committed:
Ricco Harbor

ruby
https://discord.com/channels/749042497610842152/779895315750715422/1458614269301100554

towerofnix committed:
Ai No Uta

ruby
https://discord.com/channels/749042497610842152/779895315750715422/1458614269301100554

towerofnix committed:
Mamida ga Afureta

ruby
https://discord.com/channels/749042497610842152/779895315750715422/1458614269301100554

towerofnix committed:
Cave (Treasure Cove!)

ruby
https://discord.com/channels/749042497610842152/779895315750715422/1458614269301100554

towerofnix committed:
Eternal Star, Thwomp Volcano, Toy Block Tower, Full of Danger

ruby
https://discord.com/channels/749042497610842152/779895315750715422/1458614269301100554

towerofnix committed:
Cog Building (Boss)

towerofnix committed:
Mt. Blaze

ruby
https://discord.com/channels/749042497610842152/779895315750715422/1458614269301100554

towerofnix committed:
Showdown with Cackletta!

ruby
https://discord.com/channels/749042497610842152/779895315750715422/1458614269301100554

towerofnix committed:
B-Doing Woods (World 2-1)

ruby
https://discord.com/channels/749042497610842152/779895315750715422/1458614269301100554

towerofnix committed:
ruby hivebreak credit
